frontend/src/components/Followings/FolloweesContainer.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { FollowingsComponent } from './FollowingsComponent';
+
+const msp = (state, ownProps) => {
+  return {
+    type: "Followees",
+    currentUser: state.entities.users[state.session.id],
+    channelUser: state.entities.users[state.ui.channelId],
+    users: Object.values(state.follows.currentChannelFollowings)
+  }
+}
+
+const mdp = (dispatch) => {
```

Comment:
`msp` vs `mdp` là gì vậy em?

Reply:
map state to props và map dispatch to props